rust/basic_bitcoin/README.md

```diff
@@ -27,6 +27,9 @@ For a deeper understanding of the ICP < > BTC integration, see the [Bitcoin inte
   SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/index.mdx).
   For local testing, `dfx >= 0.22.0-beta.0` is required.
 
+On Mac, `llvm` with the `wasm32-unknown-unknown` target (which is not included in the XCode installation by default) is required.
```

Comment:
Given that the other prerequisite is itemized with a bullet and a checkbox (`* [x]`), this note here about macOS is not so prominent so my impression is it could easily be overlooked. So I suggest to make this

`* [x] On macOS, `llvm` with...`

Reply:
makes sense, done